.github/workflows/ci.yml

```diff
@@ -5,12 +5,14 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, ubuntu-latest]
+        os: [macos-latest, ubuntu-latest, windows-latest]
         node: ['16', '18', '19']
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node }}
       - run: npm ci
-      - run: npm test
\ No newline at end of file
+      - run: npm run test:lint
+        if: matrix.os != 'windows-latest'
```

Comment:
This could very well be a line ending problem. I tried to fix it from the windows machine, but git didn't register any file change.